integration-test/helpers/harness.js

```diff
@@ -2,6 +2,9 @@ const puppeteer = require('puppeteer')
 const execSync = require('child_process').execSync
 
 jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000
+if (process.env.KEEP_OPEN) {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000 * 1000
```

Comment:
Some of the testing changes in develop might fix issues you were seeing here (I hope)

Reply:
This was mostly just for giving me an easier time testing a single test and keeping it open on the test I wanted so I could use the inspector etc.